target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8072-301w.dts

```diff
@@ -0,0 +1,836 @@
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_system_green: system-green {
+			label = "green:system";
+			gpios = <&tlmm 1 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_system_red: system-red {
+			label = "red:system";
+			gpios = <&tlmm 3 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_RED>;
+		};
+
+		led_pwr_green: pwr-green {
+			label = "green:pwr";
+			gpios = <&tlmm 4 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_wifi_green: wifi-green {
+			label = "green:wifi";
+			gpios = <&tlmm 42 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_lan1_green: lan1-green {
+			label = "green:lan1";
+			gpios = <&tlmm 6 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_lan1_amber: lan1-amber {
+			label = "amber:lan1";
+			gpios = <&tlmm 7 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_AMBER>;
+		};
+
+		led_lan2_green: lan2-green {
+			label = "green:lan2";
+			gpios = <&tlmm 8 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_lan2_amber: lan2-amber {
+			label = "amber:lan2";
+			gpios = <&tlmm 11 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_AMBER>;
+		};
+    
+		led_lan3_green: lan3-green {
+			label = "green:lan3";
+			gpios = <&tlmm 12 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_lan3_amber: lan3-amber {
+			label = "amber:lan3";
+			gpios = <&tlmm 13 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_AMBER>;
+		};
+    
+		led_lan4_green: lan4-green {
+			label = "green:lan4";
+			gpios = <&tlmm 14 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_lan4_amber: lan4-amber {
+			label = "amber:lan4";
+			gpios = <&tlmm 15 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_AMBER>;
+		}; 
+   
+		led_10g_1_green: 10g_1-green {
+			label = "green:10g_1";
+			gpios = <&tlmm 51 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_10g_1_amber: 10g_1-amber {
+			label = "amber:10g_1";
+			gpios = <&tlmm 52 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_AMBER>;
+		};
+
+		led_10g_2_green: 10g_2-green {
+			label = "green:10g_2";
+			gpios = <&tlmm 54 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_10g_2_amber: 10g_2-amber {
+			label = "amber:10g_2";
+			gpios = <&tlmm 56 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_AMBER>;
+		};
+	};
+
+	
+};
+
+
+&tlmm {
+
+	mdio_pins: mdio-pins {
+		mdc {
+			pins = "gpio68";
+			function = "mdc";
+			drive-strength = <8>;
+			bias-pull-up;
+		};
+
+		mdio {
+			pins = "gpio69";
+			function = "mdio";
+			drive-strength = <8>;
+			bias-pull-up;
+		};
+	};
+	
+	button_pins: button_pins {
+		wps_button {
+			pins = "gpio57";
+			function = "gpio";
+			drive-strength = <8>;
+			bias-pull-up;
+		};
+		
+		rst_button {
+			pins = "gpio67";
+			function = "gpio";
+			drive-strength = <8>;
+			bias-pull-up;
+		}; 
+	};
+
+	/*
+	 *remove qpic pins due to device is using all of the pins for leds-gpio
+	 */
+	qpic_pins: qpic-pins {
```

Comment:
Can you try if adding status = "disabled" works instead?
And if not, then delete the whole node?

Reply:
yes I can check that.

I already tried deleting that node, but that leads to an error from the dtc due to qpic-nand is using the qpic-pins. Means deleting the entire nand node would also necessary.

I'll try disabling the qpic-pins node first.

Reply:
disabling worked. Led's still working.

Reply:
Great, please use that.